h2o-core/src/main/java/water/network/SecurityUtils.java

```diff
@@ -96,15 +96,24 @@ public static String generateSSLConfig(SSLCredentials credentials) throws IOExce
         return generateSSLConfig(credentials, temp.getAbsolutePath());
     }
 
-    public static String generateSSLConfig(SSLCredentials credentials, String file) throws IOException {
+    static String generateSSLConfig(SSLCredentials credentials, String file) throws IOException {
         Properties sslConfig = new Properties();
         sslConfig.put("h2o_ssl_protocol", defaultTLSVersion());
-        sslConfig.put("h2o_ssl_jks_internal", credentials.jks.getLocation());
+        sslConfig.put("h2o_ssl_jks_internal", credentials.jks.name);
         sslConfig.put("h2o_ssl_jks_password", credentials.jks.pass);
-        sslConfig.put("h2o_ssl_jts", credentials.jts.getLocation());
+        sslConfig.put("h2o_ssl_jts", credentials.jts.name);
         sslConfig.put("h2o_ssl_jts_password", credentials.jts.pass);
         FileOutputStream output = new FileOutputStream(file);
-        sslConfig.store(output, "");
+        try {
+            sslConfig.store(output, "");
+            output.close();
+        } finally {
+            try {
+                output.close();
+            } catch (IOException e) {
+                // ignore
```

Comment:
do we really want to eat this exception?

Reply:
Yes, we can - thanks to the close() in try-statement.